deploy/kne/kind.yaml

```diff
@@ -0,0 +1,18 @@
+cluster:
+  kind: Kind
+  spec: 
+    name: kne
+    recycle: True
+    version: 0.11.1
+    image: kindest/node:v1.22.0
+ingress:
+  kind: MetalLB
+  spec:
+    manifests: ../../manifests/metallb
```

Comment:
Does this work if you do not call the CLI from the dir its built in? I was having some trouble figuring out how best to resolve paths like this

Reply:
see the code i look at the path that the yaml loaded from then use that directory as relative path from there